astroid/rebuilder.py

```diff
@@ -194,7 +190,7 @@ def visit_module(
             [self.visit(child, newnode) for child in node.body],
             doc_node=self.visit(doc_ast_node, newnode),
         )
-        if IS_PYPY and PY38:
+        if IS_PYPY:
             self._reset_end_lineno(newnode)
```

Comment:
Shouldn't we remove the condition and everything in it here as it was an AND and can never be true now ?